src/main/java/com/listywave/user/repository/user/elastic/UserElasticRepository.java

```diff
@@ -0,0 +1,101 @@
+package com.listywave.user.repository.user.elastic;
+
+import static com.listywave.common.exception.ErrorCode.ELASTICSEARCH_REQUEST_FAILED;
+import static com.listywave.common.util.PaginationUtils.checkEndPage;
+
+import co.elastic.clients.elasticsearch.ElasticsearchClient;
+import co.elastic.clients.elasticsearch._types.ElasticsearchException;
+import co.elastic.clients.elasticsearch._types.query_dsl.MatchQuery;
+import co.elastic.clients.elasticsearch._types.query_dsl.Query;
+import co.elastic.clients.elasticsearch._types.query_dsl.TermQuery;
+import co.elastic.clients.elasticsearch.core.SearchRequest;
+import co.elastic.clients.elasticsearch.core.SearchResponse;
+import com.listywave.common.exception.CustomException;
+import com.listywave.user.application.domain.UserDocument;
+import com.listywave.user.application.dto.search.UserElasticSearchResponse;
+import com.listywave.user.application.dto.search.UserElasticSearchResult;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Slice;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@RequiredArgsConstructor
+public class UserElasticRepository {
+
+    private static final float NGRAM_BOOST = 2.0f;
+    private static final float JASO_BOOST = 1.5f;
+    private final ElasticsearchClient esClient;
```

Comment:
상수와 변수 사이에 공백 한 줄 추가하는 것이 컨벤션입니다!

```suggestion
    private static final float NGRAM_BOOST = 2.0f;
    private static final float JASO_BOOST = 1.5f;
    
    private final ElasticsearchClient esClient;
```

Reply:
앗 몰랐습니다 ㅎㅎ 감사합니다!